src/helpers/ptracker-form-helpers.ts

```diff
@@ -0,0 +1,14 @@
+export const generateInfantPTrackerId = (fieldId: string, motherPtrackerId: string): string | undefined => {
+    if (!fieldId || !motherPtrackerId) return;
+  
+    return fieldId === 'infantPtrackerid'
+      ? motherPtrackerId + '1'
+      : fieldId.includes('_')
+      ? motherPtrackerId.concat(fieldId.split('_')[1])
+      : undefined;
+  };
+
+  
+  
+  
+  
```

Comment:
Please configure ur linter to run on save to remove extra lines as well as fix indenting